### PR TITLE
Build wheel for mysqlclient on MacOS

### PR DESCRIPTION
**Pull request by 9a24f0**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -34,8 +34,8 @@ jobs:
         env:
           CIBW_PROJECT_REQUIRES_PYTHON: ">=3.7, <3.10"
           CIBW_BUILD: "cp*"
-          CIBW_BEFORE_ALL_MACOS: brew install mysql gcc
-          CIBW_BEFORE_ALL_WINDOWS: choco install mysql-python
+          CIBW_BEFORE_ALL_MACOS: brew install mysql gcc openssl@1.1
+          CIBW_BEFORE_ALL_MACOS: export LIBRARY_PATH=$LIBRARY_PATH:/usr/local/opt/openssl/lib/
```
Comment (Huy-Ngo) [inline .github/workflows/build.yml]:
```suggestion
          CIBW_BEFORE_ALL_MACOS: |
            brew install mysql gcc openssl@1.1
            export LIBRARY_PATH=$LIBRARY_PATH:/usr/local/opt/openssl/lib/
```
Why not use [newer version](https://formulae.brew.sh/formula/openssl@3#default) though? Also I still don't think this is correct. I don't use mac but usually installation command install binaries/libs on /usr/bin and /usr/lib, not /usr/local/opt

**Review by Huy-Ngo — CHANGES_REQUESTED** [review]

**Comment by Huy-Ngo** [comment]
>   Warning: gcc 11.2.0 is already installed and up-to-date.
  To reinstall 11.2.0, run:
    brew reinstall gcc
  Warning: openssl@1.1 1.1.1l_1 is already installed and up-to-date.


So, openssl is already installed.

I recommend you follow the official instruction.

**Comment by Huy-Ngo** [comment]
> I recommend you follow the official instruction.

Nvm that, I don't think that work.

Maybe can you try these https://stackoverflow.com/questions/33362853/os-x-cannot-find-ssl-headers?

**Comment by Huy-Ngo** [comment]
> Warning: Refusing to link macOS provided/shadowed software: openssl@1.1
  If you need to have openssl@1.1 first in your PATH, run:
    echo 'export PATH="/usr/local/opt/openssl@1.1/bin:$PATH"' >> /Users/runner/.bash_profile
  
  > For compilers to find openssl@1.1 you may need to set:
    export LDFLAGS="-L/usr/local/opt/openssl@1.1/lib"
    export CPPFLAGS="-I/usr/local/opt/openssl@1.1/include"
  
 I suppose setting LDFLAGS should help.

**Comment by 9a24f0** [comment]
> > Warning: Refusing to link macOS provided/shadowed software: openssl@1.1
> > If you need to have openssl@1.1 first in your PATH, run:
> > echo 'export PATH="/usr/local/opt/openssl@1.1/bin:$PATH"' >> /Users/runner/.bash_profile
> 
> > For compilers to find openssl@1.1 you may need to set:
> > export LDFLAGS="-L/usr/local/opt/openssl@1.1/lib"
> > export CPPFLAGS="-I/usr/local/opt/openssl@1.1/include"
> 
> I suppose setting LDFLAGS should help.

Seemingly not LOL

**Comment by Huy-Ngo** [comment]
I hope someone who use and know Mac can help :D

**Review by Huy-Ngo — APPROVED** [review]

**Comment by Huy-Ngo** [comment]
Nice, thank you.

> We went looking everywhere, but couldn’t find those commits.

You don't have to squash the commits and force-push.

**Comment by Huy-Ngo** [comment]
Oh right, should have checked [the doc](https://cibuildwheel.readthedocs.io/en/stable/options/#environment) first :sweat_smile:

**Pull request merged**